core/controllers/cron.py

```diff
@@ -106,6 +107,22 @@ def get(self):
             activity_jobs_one_off.IndexAllActivitiesJobManager.create_new())
 
 
+class CronRecomputeStatisticsHandler(base.BaseHandler):
```

Comment:
I'm a bit confused about why this is a cron job. I thought we were only going to run the MR job in one-off cases if we detect e.g. data corruption? /cc @pranavsid98

Reply:
Yep, this is a one-off job that gets triggered in cases of data corruption. Aren't cron jobs sort of like periodic jobs?

Reply:
Yes, cron jobs get triggered at fixed times (e.g. 9 am every other day). Probably want to do a OneOffMapreduceJob instead in stats_jobs_one_off.py.